README.md

```diff
@@ -239,6 +236,17 @@ It should look something like this for ABP:
 - Click Save
 - Turn on the device and you are ready to go
 
+### Optional device properties
+
+Customizations to lora devices are set by creating specific twin desired properties on the device. The following customizations are available:
+
+|Name|Description|Configuration|When to use|
+|-|-|-|-|
+|Enable/disable downstream messages|Allows disabling the downstream (cloud to device) for a device. By default downstream messages are enabled| Add twin desired property `"Downlink": false` to disable downstream messages. The absence of the twin property or setting value to `true` will enable downlink messages.|Disabling downlink on devices decreases message processing latency, since the network server will not look for cloud to device messages when an uplink is received. Only disable it in devices that are not expecting messages from cloud|
+|Preferred receive window|Allows setting the device preferred receive window (RX1 or RX2). The default preferred receive window is 1| Add twin desired property `"PreferredWindow": 2` sets RX2 as preferred window. The absence of the twin property or setting the value to `1` will set RX1 as preferred window.|Using RX2 as preferred window will cause the network server to allocate more time when checking for cloud to device messages. The increased allocated time improves chances that cloud to device messages will arrive the lora device with fewer uplinks. The downside is that the device will only receive downlink messages in RX2. Recommended in scenarios that a cloud to device message needs to be sent as a response to an upstream message (i.e. setting current time)|
+
```

Comment:
Using second window increase the chances that the end application can process the upstream message and send a cloud to device message to the lora device without requiring and additional upstream message. Basically completing the round trip in less than 2 seconds.